native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java

```diff
@@ -0,0 +1,204 @@
+import io.ballerina.toml.syntax.tree.SyntaxTree;
+import io.ballerina.toml.syntax.tree.TableArrayNode;
+import io.ballerina.toml.syntax.tree.TableNode;
+import io.ballerina.toml.syntax.tree.Token;
+import io.ballerina.toml.validator.SampleNodeGenerator;
+import io.ballerina.tools.text.TextDocument;
+import io.ballerina.tools.text.TextDocuments;
+
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+
+import static io.ballerina.persist.PersistToolsConstants.DEFAULT_DATABASE;
+import static io.ballerina.persist.PersistToolsConstants.DEFAULT_HOST;
+import static io.ballerina.persist.PersistToolsConstants.DEFAULT_PASSWORD;
+import static io.ballerina.persist.PersistToolsConstants.DEFAULT_PORT;
+import static io.ballerina.persist.PersistToolsConstants.DEFAULT_USER;
+import static io.ballerina.persist.PersistToolsConstants.KEY_DATABASE;
+import static io.ballerina.persist.PersistToolsConstants.KEY_HOST;
+import static io.ballerina.persist.PersistToolsConstants.KEY_PASSWORD;
+import static io.ballerina.persist.PersistToolsConstants.KEY_PORT;
+import static io.ballerina.persist.PersistToolsConstants.KEY_USER;
+
+
+/**
+ * Class to create syntax tree for Config.toml.
+ */
+public class CreateSyntaxTree {
+    private static final PrintStream outStream = System.out;
+    private static String[] nodeMap = {KEY_HOST, KEY_PORT, KEY_USER, KEY_DATABASE, KEY_PASSWORD};
+    private static String[] defaultValues = {DEFAULT_HOST, DEFAULT_PORT, DEFAULT_USER, DEFAULT_DATABASE,
+            DEFAULT_DATABASE};
+
+    /**
+     * Method to create a new Config.toml file with database configurations.
+     */
+    public static SyntaxTree createToml() {
+        NodeList<DocumentMemberDeclarationNode> moduleMembers = AbstractNodeFactory.createEmptyNodeList();
+        moduleMembers = populateNodeList(moduleMembers, true);
+        Token eofToken = AbstractNodeFactory.createIdentifierToken("");
+        DocumentNode documentNode = NodeFactory.createDocumentNode(moduleMembers, eofToken);
+        TextDocument textDocument = TextDocuments.from(documentNode.toSourceCode());
+        SyntaxTree syntaxTree = SyntaxTree.from(textDocument);
+        return syntaxTree;
+    }
+
+    /**
+     * Method to update the Config.toml with database configurations.
+     */
+    public static SyntaxTree updateToml(Path configPath, String name) throws IOException {
+
+        boolean isTableEntry = false;
+        ArrayList<String> existingNodes = new ArrayList<String>();
+        NodeList<DocumentMemberDeclarationNode> moduleMembers = AbstractNodeFactory.createEmptyNodeList();
+        Path fileNamePath = configPath.getFileName();
+        TextDocument configDocument = TextDocuments.from(Files.readString(configPath));
+        SyntaxTree syntaxTree = SyntaxTree.from(configDocument, fileNamePath.toString());
+        DocumentNode rootNote = (DocumentNode) syntaxTree.rootNode();
```

Comment:
```suggestion
        DocumentNode rootNote = syntaxTree.rootNode();
```

Reply:
Removed in [e4c809d](https://github.com/ballerina-platform/persist-tools/pull/9/commits/e4c809da564408437e379bbe491d495ca2cbd2e8)